packages/algob/src/lib/tx.ts

```diff
@@ -174,7 +174,25 @@ async function mkTx (
       txIdxMap.set(index, [name, { total: 1, decimals: 1, unitName: "MOCK" }]);
       break;
     }
+    case TransactionType.ModifyAsset: {
+      // fetch asset mutable properties from network and set them (if they are not passed)
+      // before modifying asset
+      const assetInfo = await deployer.getAssetByID(txn.assetID);
+      txn.fields.manager =
+        txn.fields.manager !== "" ? txn.fields.manager ?? assetInfo.params.manager : undefined;
```

Comment:
It's more clear if we put a condition when to update. How about this:
```suggestion
      if (txn.fields.manager === "" ) txn.fields.manager = undefined;
```

Reply:
yes, thanks for suggestion. Updating.